pybossa/auth/task.py

```diff
@@ -65,6 +65,20 @@ def _only_admin_or_subadminowners(self, user, task=None, project_owners_ids=None
             project_owners_ids = project.owners_ids
         return user.admin or (user.subadmin and user.id in project_owners_ids)
 
+    def _only_admin_or_owners(self, user, task=None, project_owners_ids=None):
```

Comment:
If ```def _update(self, user, task):``` doesn't need to change then this function is not needed.

Reply:
As mentioned above, I think we need _update, hence this function also reqd.